### ISSUE TITLE
Propose interops: Neo.Const.Put and Neo.Const.Get

### ISSUE THREAD

**Issue by igormcoelho** [neo-project/neo#2040]
**Summary or problem description**
In some scenarios, such as designing your own cryptography on Neo, we may want to have access to global constants very frequently. To make this cheap and to avoid issues with changing values during stateless verifications, we propose to keep this in constant global slots.
Motivation: https://github.com/neo-project/neo-vm/issues/77

**Do you have any solution you want to propose?**
Propose interops Neo.Const.Put and Neo.Const.Get, where "Neo.Const.Put <key> <value>"  will keep value forever attached to the given key, and "Neo.Const.Get <key>" will retrieve this value.
In my opinion, it's important that "Neo.Const.Get" has some FATAL behavior, if constant doesn't exist. By preventing any type of exception handling, we are certain that behavior may never change during past verifications that didn't have the constant (but proceed anyway), and "now they suddently have". This way, I think it's quite safe for usage on stateless verifications.

GAS price can be high for Put (since it's eternal data... not different from Storage though), and very cheap on Get (could be certainly memory cached).

**Neo Version**
- Neo 2
- Neo 3

**Where in the software does this update applies to?**
- Ledger

**Comment by igormcoelho** [comment]
Note this is global, not same as PutEx const: https://github.com/neo-project/neo/issues/341

**Comment by Qiao-Jin** [comment]
Agree. And I think we also need to define corresponding regulations such as:

key/data size range

supported data type

rule of charging fee (according to data size?)

..etc

**Comment by igormcoelho** [comment]
@shargon I read your issue https://github.com/neo-project/neo/issues/1962 and maybe it's a good application for Global Const stuff... what do you think? Someone registers verification script as const, and just loads it several times from witnesses. Since Const does not offend non-deterministic behavior (that may happen with Storage), this may solve two problems at once.. what do you think?
